src/mahoji/commands/mix.ts

```diff
@@ -171,6 +187,9 @@ export const mixCommand: OSBMahojiCommand = {
 		if (boosts.length > 0) {
 			str += `\n**Boosts:** ${boosts.join(', ')}`;
 		}
+		if (hasScroll) {
+			str += ' Your scroll of cleansing enables you to save 10% of your secondaries: ';
```

Comment:
should make this end with ` secondaries.`

Reply:
Good catch, was originally planning on including a qty saved, hence the colon, but ended up not doing that. should be fixed now.